jobscheduler/job_scheduler.proto

```diff
@@ -0,0 +1,72 @@
+syntax = "proto3";
+
+option java_multiple_files = true;
+option java_package = "com.gravitational.jobscheduler";
+option java_outer_classname = "JobSchedulerServerProto";
+
+package jobscheduler;
+
+// Scheduler server methods
```

Comment:
Hint: for final version we would ask for more detailed comments according to https://github.com/golang/go/wiki/CodeReviewComments